hack/run-lint.sh

```diff
@@ -28,4 +29,19 @@ then
 fi
 
 # configured by .golangci.yml
-exec env GO111MODULE=on "$gopath/bin/golangci-lint" run
+"$gopath/bin/golangci-lint" run
+
+# install impi that ensures import grouping is done consistently
+if ! [[ -x "${gopath}/bin/impi" ]]
+then
+   echo >&2 'Installing impi'
+   tmp="$(mktemp -d)"
+   pwd="$(pwd)"
+   trap 'rm -rf -- "${tmp}"' EXIT
+   cd "${tmp}"
+   go mod init foo
+   go get github.com/pavius/impi/cmd/impi@c1cbdcb
```

Comment:
Is there a particular reason why you want to pin the exact commit version? For me,
```bash
GOPATH=/tmp/impi GO111MODULE=off GOBIN="/home/me/go/bin" go get github.com/pavius/impi/cmd/impi
```
works just fine.

Reply:
yes, works fine but they can break the master.

I thought we shouldn't break our tool when this happens.

However, since it's a dev tool, we might tolerate it and actually it will be a reminder to update the linter.

Reply:
now installing from master.